test/tst.common.js

```diff
@@ -160,6 +160,20 @@ function runTests(cons, label)
 	    '    at runTests (dummy filename)',
 	    '    at Object.<anonymous> (dummy filename)'
 	].join('\n') + '\n' + nodestack);
+
+	stack = mod_testcommon.cleanStack(VError.fullStack(new cons('Some error')));
+	mod_assert.equal(stack, [
+		stackname + ': Some error',
+		'    at runTests (dummy filename)',
+		'    at Object.<anonymous> (dummy filename)'
+	].join('\n') + '\n' + nodestack);
+
+	stack = mod_testcommon.cleanStack(VError.fullStack(new Error('Some error')));
```

Comment:
This one doesn't need to run for each class, but I suppose there's not a better place to put it at the moment.